### PR TITLE
Update the layout of package list.

### PR DESCRIPTION
**Pull request by feiling**
This contains changes for both https://github.com/NuGet/Home/issues/1491 (UI change: change class SearchResultPackageMetadata to match the new UI design) and https://github.com/NuGet/Home/issues/1431 Minor UI update: update the package list according to the new design

### CODE REVIEW

**Comment by feiling** [comment]
per feedback from yishai, the version numbers are now right aligned.

image

File: src/NuGet.Clients/PackageManagement.UI/Models/SearchResultPackageMetadata.cs [hunk 1]
```diff
@@ -104,4 +353,13 @@ public override string ToString()
             return Id;
         }
     }
-}
+
+    internal class BackgroundLoaderResult
```
Comment (MeniZalzman) [inline src/NuGet.Clients/PackageManagement.UI/Models/SearchResultPackageMetadata.cs:357]: Please move this class to its own file

File: src/NuGet.Clients/PackageManagement.UI/Models/SearchResultPackageMetadata.cs [hunk 2]
```diff
@@ -4,41 +4,297 @@
...
+        public int? DownloadCount
+        {
+            get
+            {
+                return _downloadCount;
+            }
+            set
+            {
+                _downloadCount = value;
+                UpdateAuthorAndDownloadText();
```
Comment (MeniZalzman) [inline src/NuGet.Clients/PackageManagement.UI/Models/SearchResultPackageMetadata.cs:232]: why we are mixing the Author update with the Download count? I don't see any reason we should mix the logic

**Comment by feiling** [comment]
I deleted the branch fei-ui-top, and this caused this pull request to be closed. Will send a new PR.

File: src/NuGet.Clients/PackageManagement.UI/Models/SearchResultPackageMetadata.cs (hunk 2 above)
Comment (feiling) [inline src/NuGet.Clients/PackageManagement.UI/Models/SearchResultPackageMetadata.cs:232]: In the UI design, the author and download count is displayed together like "by Microsoft, 13.5M downloads". To get this string, we have to combine the two properties.

**Pull request closed without merging**